openassessment/workflow/models.py

```diff
@@ -61,12 +61,12 @@ class AssessmentWorkflow(TimeStampedModel, StatusModel):
     STEPS = ASSESSMENT_API_DICT.keys()
 
     STATUSES = [
-        "waiting",  # User has done all necessary assessment but hasn't been
-                    # graded yet -- we're waiting for assessments of their
-                    # submission by others.
-        "done",  # Complete
+        "waiting",    # User has done all necessary assessment but hasn't been
```

Comment:
Why these changes were done?

Reply:
I added a comma after status

Reply:
the comma is great, why did you add two more spaces before the comment?

Reply:
Done